src/components/base/BaseButton.vue

```diff
@@ -76,9 +77,7 @@ export default defineComponent({
 </script>
 
 <template>
-  <!-- TODO: switch line below with the following one after implementation of ROUTING -->
-  <component :is="componentType" :class="buttonClasses" @click="this.$emit('btnClick')">
-    <!-- <component :is="componentType" :class="buttonClasses" :to="to" @click="this.$emit('btnClick')"> -->
+  <component :is="componentType" :class="buttonClasses" :to="to" @click="this.$emit('btnClick')">
```

Comment:
Wouldn't it be better to use the `click` event name here instead of `btnClick`? 🤔

Reply:
Oh, I thought that it's some kind of a reserved keyword I should not use